src/Api.php

```diff
@@ -289,6 +289,7 @@ protected function query($uri, array $data = [], $method = 'GET', array $headers
           return $this->httpClient->post($this->host . $uri, $data);
 
         case 'PURGE':
+          $data['connect_timeout'] = 3;
```

Comment:
@tomasinchoo Thanks for this PR!

Rather then specifying `3` here directly, I think this approach could be improved in one of two ways:

1. Use a service parameter to allow for end-user overrides of this value. We current do this for `fastly.host`, for example.

Defined here: https://github.com/vvuksan/fastly-drupal/blob/8.x-3.x/fastly.services.yml#L2
Passed to the service: https://github.com/vvuksan/fastly-drupal/blob/8.x-3.x/fastly.services.yml#L7
Used in the service: https://github.com/vvuksan/fastly-drupal/blob/8.x-3.x/src/Api.php#L59

2. Define a class constant to provide more clarity. Something like:

`const FASTLY_API_CONNECT_TIMEOUT = 3;`

Also, do we want to apply this to POST requests as well, @vvuksan